### PR TITLE
Update testing-questions.md

### PR DESCRIPTION
**Pull request by majonathany**
# Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New Question
- [ ] Revision of an existing question
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate) 

# Checklist:

- [x] My content follows the style guidelines of this project
- [x] I have performed a self-review of my own content

Pull requests should be thought of as a conversation. There will be some back and forth when trying to get code merged into this or any other project. With all but the simplest changes you can and should expect that the maintainers of the project will request changes to your code. Please be aware of that and check in after you open your PR in order to get your code merged in cleanly. 

Thanks!

### CODE REVIEW

File: src/questions/testing-questions.md [hunk 1]
```diff
@@ -9,3 +9,6 @@ permalink: /questions/testing-questions/index.html
 * What is the difference between a unit test and a functional/integration test?
 * What is the purpose of a code style linting tool?
 * What are some of the testing best practices?
+* What is an end to end test, and how is it different from a unit or integration test?
```
Comment (vvscode) [inline src/questions/testing-questions.md]: `* What is the difference between a unit test and a functional/integration test?` 
is it a dublicate?

Comment (majonathany) [reply src/questions/testing-questions.md]: Yes, this is. That question can be deleted.

Comment (vvscode) [reply src/questions/testing-questions.md]: so ? are you going to do that?

**Comment by vvscode** [comment]
You're welcome to reopen it when you're ready to spend time

**Pull request closed without merging**